protostar/configuration_file/configuration_file.py

```diff
@@ -17,19 +15,10 @@
 ProfileName = str
 ContractName = str
 
-
-@dataclass(frozen=True)
-class ConfigurationFileData:
-    min_protostar_version: Optional[str]
-    contract_name_to_path_str: Dict[ContractName, str]
-    lib_path_str: Optional[str]
-    command_name_to_config: CommandNameToConfig
-    shared_command_config: CommandConfig
-    profile_name_to_commands_config: Dict[ProfileName, CommandNameToConfig]
-    profile_name_to_shared_command_config: Dict[ProfileName, CommandConfig]
+TConfigurationFileModel = TypeVar("TConfigurationFileModel")
```

Comment:
Newer versions of pylint will yell here (see https://github.com/software-mansion/protostar/pull/713/commits/a07d020b88af037ef9993603e254473ff86c3b0a). Valid name would be `ConfigurationFileModelT`